tezos/context/src/working_tree/serializer.rs

```diff
@@ -471,6 +535,14 @@ impl From<StorageError> for DeserializationError {
     }
 }
 
+impl From<DBError> for DeserializationError {
```

Comment:
If possible, lets start using the `#[from]` annotation now that the `thiserror` [PR has been merged](https://github.com/tezedge/tezedge/pull/605). Why is the `Box` needed?

Reply:
@tizoc The `Box` is required because it would make an infinite recursion between `DeserializationError` and `DBError`.
It seems that `#[from]` doesn't handle this case